tamr_unify_client/models/dataset/resource.py

```diff
@@ -178,6 +178,21 @@ def from_geo_features(self, features, geo_attr=None):
             self._features_to_updates(features, record_id, key_attrs, geo_attr)
         )
 
+    def upstream_datasets(self):
+        """The Dataset's upstream datasets.
+
+        :return: A list of the Dataset's upstream datasets.
```

Comment:
Missing comment for return type (:rtype: list[:class:\`~tamr_unify_client.models.dataset.resource.Dataset\`])